juntagrico/static/juntagrico/css/juntagrico.css

```diff
@@ -374,14 +379,19 @@ th {
 
 #managementlist table {
     width: 100%;
-    font-size: 14px;
+    font-size: 12pt;
 }
 
 #managementlist table th {
     vertical-align: top;
     padding-left: 0;
 }
 
+#managementlist a:not(.btn) {
+    color: #408440;
```

Comment:
#408040
since it has the same values for cmyk as the original green only the k value is higher